### PR TITLE
jackett: 0.18.925 -> 0.19.147

### LINKED ISSUE #143247: jackett: 0.18.95 -> 0.19.71 + Standalone
###### Checklist

- [ ] Copy the code already made from the link down below
- [ ] Fix and tweak as you like to fit nixpkgs standard

###### Project name
`nix search` name: jackett

current version: 0.18.95
desired version: 0.19.71

###### Notify maintainers

maintainers: @edwtjo @nyanloutre @purcell 

###### Note for maintainers

I already did that but I have no time to make a PR. Everything is available here, from the hash update to the migration from Mono to the standalone version : https://gitlab.com/NicolasGuilloux/nixos-configuration/-/blob/master/pkgs/jackett/default.nix

Cheers guys!

### PR DESCRIPTION
**Pull request by flexagoon**
###### Motivation for this change
- Update Jackett to the latest version
- Use standalone versions of Jackett instead of Mono versions

Closes #143247

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by NicolasGuilloux — CHANGES_REQUESTED** [review]

File: pkgs/servers/jackett/default.nix [hunk 1]
```diff
@@ -1,30 +1,82 @@
...
   '';
 
   meta = with lib; {
     description = "API Support for your favorite torrent trackers";
     homepage = "https://github.com/Jackett/Jackett/";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ edwtjo nyanloutre purcell ];
-    platforms = platforms.all;
+    maintainers = with maintainers; [ edwtjo nyanloutre purcell flexagoon ];
+    platforms = with platforms; [ "x86_64-linux" "aarch64-linux" "armv7a-linux" "armv7l-linux" "x86_64-darwin" ];
```
[the remark this answers is not in the document]
Comment (flexagoon) [reply pkgs/servers/jackett/default.nix:76]: @SuperSandro2000

File: pkgs/servers/jackett/default.nix [hunk 2]
```diff
@@ -1,30 +1,82 @@
...
+      fetchurl
+        {
+          url = "https://github.com/Jackett/Jackett/releases/download/v${version}/Jackett.Binaries.macOS.tar.gz";
+          sha256 = "1489pgp1y77crpnnm5mfwvz81400hm3cnv0fq753wfpmgm9g4d4q";
+        }
+    else
+      fetchurl {
+        url = "https://github.com/Jackett/Jackett/releases/download/v${version}/Jackett.Binaries.LinuxAMDx64.tar.gz";
+        sha256 = "0dfxnkql7l6dpyqiwd9al8a0nr93s5avb7k7pj0mlq8q38lb8bfk";
+      };
```
[the remark this answers is not in the document]
Comment (flexagoon) [reply pkgs/servers/jackett/default.nix]: @SuperSandro2000

**Comment by GGG-KILLER** [comment]
I'd also like to highlight that this fixes the following issue:
```
Ssl error:1000007d:SSL routines:OPENSSL_internal:CERTIFICATE_VERIFY_FAILED
```
related to the expiration of Let's Encrypt intermediate certificate.

**Comment by IvarWithoutBones** [comment]
I've opened #144563, which makes jackett compile from source + updates the package to it's current latest version. I thought you all might be interested :)

**Comment by GGG-KILLER** [comment]
Question about both this and #144563.

Shouldn't we keep a `jackett-mono` package for environments that aren't supported by .NET? Or is there no system that Mono and Nix support that .NET doesn't?

File: pkgs/servers/jackett/default.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/servers/jackett/default.nix:76]: It should be ``platforms.linux``.

**Comment by flexagoon** [comment]
I'm closing this due to #144563 being merged

**Pull request closed without merging**